### DISCUSSION TITLE
Support for Jellyfin Logins on Jellyseerr

### DISCUSSION THREAD

**Discussion by milesjniemeyer** [9p4/jellyfin-plugin-sso#194] [Q&A]
Is there anyway to be able to use the jellyfin-plugin-sso to enable users to login to Jellyseerr considering Jellyseerr allows users to login with their Jellyfin account? If not, what would be a good, streamlined solution that doesn't require manually creating Jellyseerr accounts?

**Comment by 9p4** [comment]
You'll have to set up a fallback provider since it seems like Jellyseerr uses direct API access to log into Jellyfin.

**Reply by milesjniemeyer** [reply]
How would I go about doing that?

**Reply by 9p4** [reply]
The TL;DR is that you'll have to set the fallback provider in the config to be `Jellyfin.Server.Implementations.Users.DefaultAuthenticationProvider`. This will allow for traditional username/password logins while keeping SSO. Each user will have to reset their password in Jellyfin, as this defaults to setting a long, random password.

**Reply by gpalsson** [reply]
> The TL;DR is that you'll have to set the fallback provider in the config to be `Jellyfin.Server.Implementations.Users.DefaultAuthenticationProvider`. This will allow for traditional username/password logins while keeping SSO. Each user will have to reset their password in Jellyfin, as this defaults to setting a long, random password.

hmm. got keycloak and your excellent sso plugin to work, but it's still a mystery how to get jellyseer to work with sso. I have tried resetting the password for the passwordless SSO account from an admin account, but it didn't make a difference and I am lost where to set the Jellyfin.Server.Implementations.Users.DefaultAuthenticationProvider in a config on the jellyfin server or what value it should hold.
